scripts/db-heal-test.py

```diff
@@ -0,0 +1,263 @@
+import argparse
+import os
+import subprocess
+import tempfile
+import time
+import shutil
+from pathlib import Path
+import sys
+
+parser = argparse.ArgumentParser(prog="DB HEAL TEST SCRIPT",
+                                 description="This script as serves as a test tool for 'db-heal' feature."
+                                             "It tests recover and LiveDB export/import.",
+                                 usage="To run this script, please provide Aida root using --aida and path to "
+                                       "AidaDb using --aida-db.\n"
+                                       "Example: python3 scripts/db-heal-test.py --aida /Path/To/Aida --aida-db"
+                                       "/Path/To/Aida/Db")
+
+# --- Parameters --- #
+parser.add_argument('--aida', type=str, help="Path to Aida root.")
+parser.add_argument('--aida-db', type=str, help="Path to AidaDB.")
+parser.add_argument("--tmp", type=str, help="Path to tmp dir.")
+parser.add_argument("--iter", type=int, help="Number of iterations.", default=1000)
+parser.add_argument("--window", type=int,
+                    help="Delay between start of sync process and forced termination (in seconds).", default=5)
+parser.add_argument("--cp-granularity", type=int,
+                    help="How often will Carmen create checkpoints (in blocks).", default=10)
+
+args = parser.parse_args()
+
+aida_path = args.aida
+aida_db_path = args.aida_db
+tmp_path = args.tmp
+number_of_iterations = args.iter
+window = args.window
+checkpoint_granularity = args.cp_granularity
+
+# Mark first checkpoint
+latest_checkpoint = checkpoint_granularity
+
+if not aida_path or aida_path == "":
+    print("please set Aida using --aida")
+    sys.exit(1)
+if not aida_db_path or aida_db_path == "":
+    print("please set AidaDB using --aida-db")
+    sys.exit(1)
+if not tmp_path or tmp_path == "":
+    tmp_path = tempfile.gettempdir()
+    print(f"tmp not set - using default {tmp_path}")
+
+# Block variables
+first_block = 0
+last_block = 60000000
+
+carmen_root = os.path.abspath('../go')
+
+# --- Script --- #
+
+# Create working dir which gets deleted after the run
+working_dir = os.path.join(tmp_path, 'db-heal-test')
+if os.path.exists(working_dir):
+    shutil.rmtree(working_dir)
+os.makedirs(working_dir)
+
+# Log file path from which we read output to find kill_block
```

Comment:
I think these files should be stored also in `tmp`, this way they will get cleared-up. At the moment they persist in current dir.